### PR TITLE
Remediations wizard progress step

### PR DESCRIPTION
**Pull request by fhlavac**
https://issues.redhat.com/browse/RHCLOUD-13720
https://marvelapp.com/prototype/gbhbja0/screen/79278089

- added a new Progress step at the end of the wizard
- a bit of code cleaning
- enhanced tests

loading
error
success

cc: @katierik

### CODE REVIEW

**Review by Hyperkid123 — APPROVED** [review]
:100:

File: packages/remediations/src/utils/utils.js [hunk 1]
```diff
@@ -155,17 +142,28 @@ export const submitRemediation = (formValues, data, basePath) => {
...
-    }
+
+    const { id } = formValues[EXISTING_PLAYBOOK] || {};
+    const isUpdate = formValues[EXISTING_PLAYBOOK_SELECTED];
+
+    (isUpdate &&
+    api.patchRemediation(id, { add, auto_reboot: formValues[AUTO_REBOOT] }, basePath) ||
+    api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)).then(({ id }) => {
+        setState({ id, percent: 100 });
+        /*data.onRemediationCreated({
```
Comment (Hyperkid123) [inline packages/remediations/src/utils/utils.js:159]: Can we remove this comment?

Comment (fhlavac) [reply packages/remediations/src/utils/utils.js:159]: Thanks @Hyperkid123 for catching that, it was a debug comment, I'll remove it.

**Comment by katierik** [comment]
Looks good!

On the success message, the word "Playbook" in the button should be lowercase "playbook" -- We only should capitalize it if it is with Ansible -- e.g. "Ansible Playbook" or only "playbook"
The external link icon is in correct - we want fa-external-link-alt

Otherwise 👍

**Comment by fhlavac** [comment]
@katierik fixed issues and updated the success variant screenshot. Thank you :-)

**Pull request merged**